cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -79,6 +79,12 @@ in the staging namespace, belonging to the purple cluster, with the label owner=
 
 ## Connecting cluster-autoscaler to Cluster API management and workload Clusters
 
+> [!IMPORTANT]
+> `--cloud-config` is the flag to configure the cloud provider. Here the cloud provider is "Cluster API".
+
+> [!IMPORTANT]
+> `--kubeconfig` is the flag to configure the cluster to watch for the need of autoscaling.
```

Comment:
`--kubeconfig` configures a mount volume path to the workload cluster KUBECONFIG for the cluster-autoscaler runtime when not running cluster-autoscaler on the workload cluster itself.

Reply:
I know the above are mouthfuls, thoughts @elmiko ?

Reply:
similar to the above, i like Jack's direction. if we are going to add these advisories i want them to specifically mention the kubeconfig. eg

```
`--kubeconfig` is the flag for specifying a mount volume  path to the kubernetes configuration (ie KUBECONFIG) to the cluster-autoscaler for communicating with the cluster-api workload cluster for the purpose of watching Nodes and Pods. This flag can be affected by the desired topology for deploying the cluster-autoscaler, please see the diagrams below for more information.
```

Reply:
```suggestion
> ``--kubeconfig` is the flag for specifying a mount volume  path to the kubernetes configuration (ie KUBECONFIG) to the cluster-autoscaler for communicating with the cluster-api workload cluster for the purpose of watching Nodes and Pods. This flag can be affected by the desired topology for deploying the cluster-autoscaler, please see the diagrams below for more information.
```